### ISSUE TITLE
Typesense AWS EC2 Errors

### ISSUE THREAD

**Issue by francesco-hayes** [typesense/typesense#668]
## Description

I currently just got an AWS EC2 instance configured and started the other day. Today, I installed typesense via the Ubuntu DEB package, which should do most of the configuration for you, from what I understand. However, I receive two errors on start. I am unsure if there is something else I need to configure on my server or not... Any guidance is appreciated. Thank you!

## Steps to reproduce
Here is the config file:
![Screen Shot 2022-07-21 at 12 05 56 PM](https://user-images.githubusercontent.com/44263980/180261513-29708f1d-b9be-43ac-ab74-d5feca8aa79d.png)

I then run the following command:
```
sudo systemctl status typesense-server.service
```
This is the result:
![Screen Shot 2022-07-21 at 12 02 53 PM](https://user-images.githubusercontent.com/44263980/180260989-87ea7aeb-d356-4345-a19b-00ee844b5434.png)

## Expected Behavior

I should see a successful creation of the service.

## Actual Behavior
The following result is the behavior with no way of connecting to the public IP for the server /health to check the status.

![Screen Shot 2022-07-21 at 12 02 53 PM](https://user-images.githubusercontent.com/44263980/180260989-87ea7aeb-d356-4345-a19b-00ee844b5434.png)

## Metadata

**Typsense Version**: 0.23.1

**OS**: Ubuntu

**Comment by jasonbosco** [comment]
@francesco-hayes When started Typesense will initially report a status of unhealthy and then after a few seconds it should become healthy. If you open the log file under `/var/log/typesense/` you should see the latest logs from the server. Could you post the last say 100 lines from the log file? 

Also, what happens if you do `curl localhost:8108/health`?

**Comment by francesco-hayes** [comment]
> @francesco-hayes When started Typesense will initially report a status of unhealthy and then after a few seconds it should become healthy. If you open the log file under `/var/log/typesense/` you should see the latest logs from the server. Could you post the last say 100 lines from the log file?
> 
> Also, what happens if you do `curl localhost:8108/health`?

Yeah, I just checked and it is returning an "{"ok": true}"

However, why is not connecting to the EC2 Public IP address?

**Comment by jasonbosco** [comment]
The default bind address is `0.0.0.0`, so Typesense will bind to all available interfaces/IPs. 

In AWS, could you make sure you've allowed access to port 8108 (which is the default Typesnese port) in the security group for the EC2 instance?

**Comment by francesco-hayes** [comment]
> 

Yes, the security group has a port range of "All" for outbound connections under that destination.

**Comment by jasonbosco** [comment]
The inbound rules should also allow access to port 8108 from your IP.

**Comment by lBroth** [comment]
Hi I have the same problem
